exporter/alertmanagerexporter/README.md

```diff
@@ -28,7 +28,7 @@ The following settings are optional:
 - `timeout` `sending_queue` and `retry_on_failure` settings as provided by [Exporter Helper](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/exporterhelper#configuration)
 - [HTTP settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/confighttp/README.md)
 - [TLS and mTLS settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md)
-- `generator_url` is the source of the alerts to be used in Alertmanager's payload and can be set to the URL of the opentelemetry collector if required
+- `generator_url` is the source of the alerts to be used in Alertmanager's payload. The default value is <http://example.com/otelcol>, and can be set to the URL of the opentelemetry collector. 
```

Comment:
I would rather set the default to `opentelemetry-collector` if an URL isn't required, or simply `<unset-otelcol>`. Using `example.com` seems out of place here.

Reply:
Thank you, I have updated the default to "opentelemetry-collector" now.